waku/v2/node/wakunode2_setup_metrics.nim

```diff
@@ -73,4 +68,8 @@ proc startMetricsLog*() =
     discard setTimer(Moment.fromNow(30.seconds), logMetrics)
   
   discard setTimer(Moment.fromNow(30.seconds), logMetrics)
+
+  # Start protocol specific metrics logging
+  when defined(rln) or defined(rlnzerokit):
+    startRlnMetricsLog()
```

Comment:
Maybe I'm missing something obvious, but why not log the RLN metrics with the other metrics? I.e. on the same timer as above (under `logMetrics`), you just do:

```suggestion
  when defined(rln) or defined(rlnzerokit):
    logRlnMetrics()
```

Reply:
In other words, I don't see the need for a separately timed task

Reply:
Agreed, will fix soon

Reply:
Addressed in [2a805e0](https://github.com/status-im/nwaku/pull/1262/commits/2a805e06ed02f591490bdfb7010353aed935e0bd)

Reply:
Maybe some protocols would like tighter control on how often their metrics are logged, that's why I've retained the `setTimer` for the rlnMetrics as well.

Reply:
Right, I think I don't quite see the need for this flexibility if we don't have any current reason for logging metrics for different protocols at different intervals. In my mind this just creates unnecessary complexity in the logic (and the possibility of somewhat confusing logs)? My pattern is to always go for the very simplest thing - this is easy to change to something a bit more intricate in future if the need arise, but changing complex code back to something simpler is much more difficult.